model/src/main/kotlin/utils/SimplePackageConfigurationProvider.kt

```diff
@@ -99,3 +102,5 @@ private fun Collection<PackageConfiguration>.checkAtMostOneConfigurationPerIdAnd
                 "${configurationsWithSameMatcher.keys.joinToString(prefix = "  ", separator = "\n  ")}."
     }
 }
+
+private fun Identifier.stripVersion(): Identifier = Identifier(type, namespace, name, "")
```

Comment:
Maybe `= copy(version = "")`?

Reply:
Actually, it is really worth having an extension function for this? I don't believe so, and would rather suggest a code comment if you feel like explaining what this line does.

Reply:
Little less repetitive and more speaking vs. fewer lines of code. Close to a draw IMO. Let me ditch it.